packages/connectivity/src/scp-cf/environment-accessor.ts

```diff
@@ -177,7 +177,7 @@ export function getEnvironmentVariable(
   if (process.env[name]) {
     return process.env[name];
   }
-  logger.info('Environment variable ' + name + ' is not defined.');
+  logger.warn('Environment variable ' + name + ' is not defined.');
```

Comment:
[nth] The format of the line 171 and line 172 seems to be strange. Could you check locally?

Reply:
[nth] Also you could take the opportunity to change the `+` to string template.